python/tvm/meta_schedule/testing/relay_workload.py

```diff
@@ -15,13 +15,230 @@
 # specific language governing permissions and limitations
 # under the License.
 """Workloads in Relay IR"""
+from enum import Enum
 from typing import Dict, Tuple
 
-import tvm.relay.testing  # pylint: disable=unused-import
 from tvm import relay
 from tvm.ir import IRModule
 from tvm.runtime import NDArray
 
+# Model types supported in Torchvision
+class MODEL_TYPE(Enum):  # pylint: disable=invalid-name
```

Comment:
I don't think we should define this huge chunk of models on mainline, at least not for now :-) Let's find alternatives

Reply:
Hey, I simplied most of the script, does the script look good to you?

Reply:
Looks great!